CorpusSearch/Extensions/NormalizationExtensions.cs

```diff
@@ -29,20 +30,28 @@ public static string RemoveDiacritics(this string target)
 
     public static string NormalizeMicrosoftWordQuotes(this string buffer)
     {
-        if (buffer.IndexOf('\u2013') > -1) { buffer = buffer.Replace('\u2013', '-'); }
-        if (buffer.IndexOf('\u2014') > -1) { buffer = buffer.Replace('\u2014', '-'); }
-        if (buffer.IndexOf('\u2015') > -1) { buffer = buffer.Replace('\u2015', '-'); }
-        if (buffer.IndexOf('\u2017') > -1) { buffer = buffer.Replace('\u2017', '_'); }
-        if (buffer.IndexOf('\u2018') > -1) { buffer = buffer.Replace('\u2018', '\''); }
-        if (buffer.IndexOf('\u2019') > -1) { buffer = buffer.Replace('\u2019', '\''); }
-        if (buffer.IndexOf('\u201a') > -1) { buffer = buffer.Replace('\u201a', ','); }
-        if (buffer.IndexOf('\u201b') > -1) { buffer = buffer.Replace('\u201b', '\''); }
-        if (buffer.IndexOf('\u201c') > -1) { buffer = buffer.Replace('\u201c', '\"'); }
-        if (buffer.IndexOf('\u201d') > -1) { buffer = buffer.Replace('\u201d', '\"'); }
-        if (buffer.IndexOf('\u201e') > -1) { buffer = buffer.Replace('\u201e', '\"'); }
-        if (buffer.IndexOf('\u2026') > -1) { buffer = buffer.Replace("\u2026", "..."); }
-        if (buffer.IndexOf('\u2032') > -1) { buffer = buffer.Replace('\u2032', '\''); }
-        if (buffer.IndexOf('\u2033') > -1) { buffer = buffer.Replace('\u2033', '\"'); }
+        //var quotes = new[] { ('\u2013', '-'),('\u2014', '-') };
+        var qmap = new Dictionary<char, char>
+        {
+            { '\u2013', '-' },
+            { '\u2014', '-' },
+            { '\u2015', '-' },
+            { '\u2017', '_' },
+            { '\u2018', '\'' },
+            { '\u2019', '\'' },
+            { '\u201a', ',' },
+            { '\u201b', '\'' },
+            { '\u201c', '\"' },
+            { '\u201d', '\"' },
+            { '\u201e', '\"' },
+            { '\u2032', '\'' },
+            { '\u2033', '\"' }
+        };
+        foreach (var key in qmap.Keys)
+            if (buffer.IndexOf(key) > -1)
```

Comment:
Please use braces unless it's a guard clause

Reply:
You mean for both the for and the if?

Reply:
I assume definitely for the if
And most likely for the for as well

Reply:
Note: cleanup in Rider  actually removes the braces

Reply:
Okay, if I just do reformat but no syntax style it will leave them